chapters/en/slides/module7/module7_01.md

```diff
@@ -0,0 +1,162 @@
+---
+type: slides
+---
+
+# What is Jupyter?
+
+---
+
+# Project Jupyter
+
+<br>
+
+<img src="/module7/proj_jupyter.png" width="200"></img>
+
+Main goals of [Project Jupyter](https://jupyter.org/):
+
+- Providing an ecosystem for **interactive** data science projects
+
+- Being **language agnostic**: support most popular programming languages
+
+<br>
+
+<p style="color:black; font-size:130%; background-color:floralwhite;">
+The name "Jupyter" -> "Julia" + "Python" + "R"
+</p>
+
+Notes:
+
+Project Jupyter was born from the IPython project in 2014.
+
+The name "Jupyter" is a combination of "Julia" (popular scientific computing language), "Python", and "R" (popular statistical analysis tool).
+
+---
+
+# JupyterLab
+
+<br>
+
+<img src="/module7/jlab.png" width="400"></img>
+
+<br>
+
+- JupyterLab is a web-based interactive development environment
+
+- You can work with various types of documents in JupyterLab
+
+- The main document type used with JupyterLab is **Jupyter notebook**
+
+Notes:
+
+JupyterLab enables you to work with documents and activities such as Jupyter notebooks, text editors, terminals, and custom components in a flexible, integrated, and extensible manner. 
+
+The main document type that is most commonly used in JupyterLab is Jupyter **notebook**.
+
+Jupyter notebooks enable you to combine narrative with interactive, executable code and its output all in one place. 
+
+We'll talk about Jupyter notebooks in more detail in the upcoming sections.
+
+---
+
+<img style="border:1px solid black;" src="/module7/jl-interface.png" width="900"></img>
+
+Notes:
+
+By now, this interface should look familiar to you!
+
+You have used the terminal and Git extension inside JupyterLab in previous chapters. In this chapter, we will go in more detail about JupyterLab's functionality, and how we can effectively use this environment for interactive development of our data science projects.
+
+---
+
+# Install and run JupyterLab
+
+<br>
+
+- To install JupyterLab, run the following command in a terminal:
+
+
+```
+conda install -c conda-forge jupyterlab
+```
+
+When asked to proceed, press `Enter`.
+
+<br>
+
+- When installation is complete, you can run:
+
+```
+jupyter lab
+```
+
+to launch JupyterLab.
+
+Notes:
+
```

Comment:
Good point. Since the whole chapter is about JupyterLab, it might be nice to have this one slide here, but not spend much time on it. I'm also open to removing it as well, because students should have installed and used JupyterLab at this point.